src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -1013,6 +1012,7 @@ class Fparser2Reader():
         ('**', BinaryOperation.Operator.POW),
         ('==', BinaryOperation.Operator.EQ),
         ('.eq.', BinaryOperation.Operator.EQ),
+        ('.eqv.', BinaryOperation.Operator.EQV),
```

Comment:
I added NEQV and XOR did some tests for the precedence of them. Notably XOR is a "legacy-extension", so fparser can't parse it at the moment. I've left it in in case we decide to support it in future but I can remove it if you'd prefer.